lib/charms/prometheus_k8s/v0/prometheus_scrape.py

```diff
@@ -1306,6 +1306,7 @@ def __init__(
         relation_name: str = DEFAULT_RELATION_NAME,
         jobs=None,
         alert_rules_path: str = DEFAULT_ALERT_RULES_RELATIVE_PATH,
+        disable_forwarding_alert_rules: bool = False,
```

Comment:
- Update PR description accordingly?
- Should we rename it to `forward_alert_rules`? `enable=True` seems easier than `disable=False`.
- We do not have a [`, *,`](https://peps.python.org/pep-3102/) here, so this new arg must  be last. Should probably add a `*,` before it.

Reply:
Done